### PR TITLE
Add initial example of a Makefile with common command targets.

### PR DESCRIPTION
**Pull request by achton**
Start with the `Makefile` here: https://github.com/reload/proref/pull/1/files#diff-b67911656ef5d18c4ae36cb6741b7965

Then check `docker-compose.yml`. The rest is cruft and scaffolding.

See also https://github.com/reload/lokalbolig/pull/109

Drawing is here: https://docs.google.com/drawings/d/1NSLA57LIXNLTR19dBGCN-eIDrOzPo8sMKyt9qqyk5b8/edit

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Example file for a Drupal-based setup.
```
Comment (rasben) [inline docker-compose.yml]: Måske omdøb filen til docker-compose.example.yml :)

**Review by rasben — APPROVED** [review]
👍 En mini ting, ellers er det good to go, så få det merge :)

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+# Run Drupal Console with an arbitrary set of parameters.
+# 	Example: make drupal-run CMD=site:status
+drupal-run: ensure-fpm
+	docker-compose exec fpm drupal ${CMD}
+
+# Run Drush with an arbitrary set of parameters.
+# 	Example: make drush-run CMD=status
+drush-run: ensure-fpm
+	docker-compose exec fpm drush ${CMD}
```
Comment (kasperg) [inline Makefile]: This target seems to be repeated?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  web:
+    image: reload/drupal-apache-fpm
+    ports:
+      - '80'
+    depends_on:
+      - fpm
+    volumes:
+      - './:/var/www/web'
+
+  fpm:
```
Comment (kasperg) [inline docker-compose.yml]: This is purely form but if we are to execute Drush and other PHP based CLI tools from within this image then I would like to see it named `php`. Running `fpm` is just one of its' use cases.

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+# The commands are:
+#   make reset 	- stop all containers, update them and db dumps and start again
+#   make up 	- just bring up all containers
+#   make stop 	- stop all containers
+#   make test	- run any local tests
+#   make logs   - follow docker logs from the containers
+#   make status - show status of the environment
+
+# List of containers
+DOCKER_CONTAINERS = web fpm db mailhog
```
Comment (kasperg) [inline Makefile]: Why do we need to keep a separate list of container?

As I understand it running `docker-compose` commands without naming containers should apply the command to all containers.

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+    image: reload/drupal-php7-fpm:7.1
```
Comment (kasperg) [inline docker-compose.yml]: Why PHP 7.1? PHP 7.2. is the recommended version to use with Drupal 8.5.

**Comment by kasperg** [comment]
If possible it would be nice to have a CircleCI setup which ran through a typical usage scenario with these commands to ensure integrity as we go along.

Also: How about moving the images to a generated diagram instead of the (nicely drawn) PNG. It is easier to maintain.

File: docker-compose.yml (hunk 1 above)
Comment (achton) [reply docker-compose.yml]: @rasben Good point, men hele repo'et er et eksempel (proref = "Project reference"), så tænker det er lidt omsonst. Jeg ville gerne gøre det muligt at køre projektet up for realz uden videre, efter man har klonet det.

File: Makefile (hunk 4 above)
Comment (achton) [reply Makefile]: Yes, this comes from a use case where the Makefile targets are split into groups of containers that can be reset independently from each other (eg. a Drupal + React setup). But it's a bit specialized and out of scope for this example, so I removed it.

File: Makefile (hunk 2 above)
Comment (achton) [reply Makefile]: One is for running Drupal Consle, the other for Drush ;)

File: docker-compose.yml (hunk 3 above)
Comment (achton) [reply docker-compose.yml]: Agreed - fixed!

File: docker-compose.yml (hunk 5 above)
Comment (achton) [reply docker-compose.yml]: Yes, fixed.

File: Makefile (hunk 2 above)
Comment (kasperg) [reply Makefile]: Gah :+1:

**Comment by achton** [comment]
I've created issues for your two remaining comments @kasperg - I will merge this if you have no objections?

**Review by kasperg — APPROVED** [review]

**Comment by kasperg** [comment]
@achton :+1: - I did not consider my comments blockers.

**Pull request merged**